vipps-report-api-faq.md

```diff
@@ -33,6 +34,49 @@ We are still quite early in the planning and design phase and will use the
 draft documentation here to discuss with potential users of the API
 and make sure we are developing the right solution.
 
+## Which API keys give access to the API?
+
+In short: The following API keys give access to the Report API:
+* The merchant's own API keys (the same API keys that are used to make payments)
+* Partner keys (the same API keys that are used to make payments)
+* Accounting partners' API keys (after the merchant has given the accounting partner access)
+
+### The merchant's own API keys
+
+The merchant's own API keys give full access to the Report API.
+
+If the merchant uses an integration partner (see
+[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types)),
+it is the same as using the merchant's own API keys.
+
+When a merchant shares its API keys for a sale unit with an integration partner,
+Vipps has no way of knowing whether the API calls are made by the merchant or
+the integration partner.
+This is why we require that all API requests made by a partner on behalf of a
+merchant are done using the partner's own API keys (the partner keys).
+
+### Partner keys
+
+If the merchant has a sale unit that is configured with a platform partner (see
+[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types)),
+the platform partner can use
+[partner keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/partner-keys)
+to make requests to the Report API.
```

Comment:
```suggestion
to make requests to the Report API. This allows them to get information about payments and transactions associated with the ledger(s) they have access to, but does not give allow them to make payments or move money in any way.
```

Reply:
but does not `give allow` them to make payments or move money in any way. One or the other?

Reply:
oops! editing error